src/components/Post/Comments/Styles.ts

```diff
@@ -6,6 +6,8 @@ import UButton from 'components/Button';
 import {InlineInput as UInlineInput} from 'components/FormElements';
 import {colors} from 'styles';
 
+import {breakpoints} from 'styles';
```

Comment:
Combine these imports into a single line:
```
import {breakpoints, colors} from 'styles';
```

Reply:
Oh, sorry about that.
let me do it now!